acceptance/install/config_test.go

```diff
@@ -0,0 +1,72 @@
+package acceptance_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/epinio/epinio/acceptance/helpers/epinio"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Epinio Installation", func() {
+	var (
+		flags      string
+		configFile string
+	)
+
+	epinioBinary := fmt.Sprintf("../../dist/epinio-%s-%s", runtime.GOOS, runtime.GOARCH)
+	epinioHelper := epinio.NewEpinioHelper(epinioBinary)
+	epinioUser := "epinio"
+	epinioPassword := "password"
+
+	BeforeEach(func() {
+		input, err := ioutil.ReadFile("../../assets/tests/config.yaml")
+		Expect(err).NotTo(HaveOccurred())
+		f, err := ioutil.TempFile("", "config")
+		Expect(err).NotTo(HaveOccurred())
+
+		configFile = f.Name()
```

Comment:
NOTE: next time we do the same (on another test), we should move it to a helper function.